### PR TITLE
homebannerupdate

### PR DESCRIPTION
**Pull request by infanta-deriv**
This PR has update of script for homebanner to include awards check and Trust Pilot
Card - https://app.clickup.com/t/20696747/DERC-2466

### CODE REVIEW

File: cypress/e2e/smoke/allHomeBanner.cy.js [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
     const urlToMatch = baseUrl.endsWith('/') ? baseUrl.slice(0, -1) : baseUrl
-
-    cy.c_visitResponsive(Cypress.env('RegionROW'))
     cy.findByRole('heading', { name: 'Trading for anyone. Anywhere. Anytime.' }).should('be.visible')
-    cy.findByText('Trade CFDs and options on global financial markets, all in one place with 24/7 trading and worldwide support.').click();
     cy.findByRole('img', { name: 'Trustpilot' }).click();
+
+    if(region === 'ROW')
+    {
+    cy.findByRole('img', { name: 'Best forex spreads APAC 2023 by UF Awards' }).should('be.visible')
```
Comment (markodonnell-deriv) [inline cypress/e2e/smoke/allHomeBanner.cy.js:12]: Indentation needed

File: cypress/e2e/smoke/allHomeBanner.cy.js [hunk 2]
```diff
@@ -1,17 +1,46 @@
...
+    cy.findByRole('img', { name: 'Broker of the year 2023 by FinanceFeeds' }).should('be.visible')
+    cy.findByRole('img', { name: 'Most trusted broker APAC 2023 by UF Awards' }).should('be.visible')
+    }
+
+    else
+    {
+    cy.findByRole('img', { name: 'Best forex spreads APAC 2023 by UF Awards' }).should('not.exist')
```
Comment (markodonnell-deriv) [inline cypress/e2e/smoke/allHomeBanner.cy.js:19]: Indentation needed

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
Indentation needed

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**